services/siren/src/Lib.hs

```diff
@@ -157,9 +157,16 @@ deleteScenario conn token userId authRole scID = do
 recoverScenario :: Connection
                 -> Int64 -- ^ token (callID)
                 -> ScenarioId -- ^ ScID (scenario id)
+                -> Maybe Int64 -- ^ User Id
```

Comment:
Minor thing: I tried to preserve the order of similar parameters in all the functions:
Connection, token, userId, authRole, ...all others...